### ISSUE TITLE
Make additional text display well for radio button and checkbox fields

### ISSUE THREAD

**Issue by shanear** [18F/web-design-standards#1082]
**Description**
For additional text like "Required" and "Optional" on a form field, it works great for text fields, but for radio button or checkbox fields, the alignment doesn't look great.

**Ask**
Make `.usa-additional_text` work with fieldset legends, used by radio button and checkbox fields.

**Additional Info**
Additional text on a text field:
<img width="485" alt="screenshot 2016-03-25 12 04 57" src="https://cloud.githubusercontent.com/assets/1596259/14048112/8524e8c0-f282-11e5-87e9-5568d747481d.png">

Additional text (as it stands) on a legend for a set of radio buttons:
<img width="474" alt="screenshot 2016-03-25 12 04 53" src="https://cloud.githubusercontent.com/assets/1596259/14048113/86ccbb30-f282-11e5-89d4-692eba6e72d7.png">

**Comment by maya** [comment]
Thanks @shanear! That's a good idea. When developing the `.usa-additional_text` class we only considered its use on input fields, not other form controls. 

I've added this as a feature request. We'll see how this fits in with our product roadmap: https://github.com/18F/web-design-standards/wiki/Product-Roadmap and component prioritization: https://github.com/18F/web-design-standards/wiki/Component-prioritization:-The-factors-that-guide-our-prioritization-process

Cc'ing @carodew.

**Comment by KHarshawat** [comment]
Hey there! As you guys are developing a more comprehensive solution to this you might want to take a look the following findings:

Initially we tried just using a red asterisk like the screenshot below:

![image](https://cloud.githubusercontent.com/assets/6335546/14115494/3da7ad50-f5a9-11e5-9c26-b0d1920c576f.png)

While we found that the red was somewhat successful in drawing people's attention, we felt the USWDS use of the word "required" would be better. As Shane mentioned, USWDS doesn't currently support non-fixed width questions like those with radio buttons or check boxes. I came up with the following solution which we'll be customizing for now all questions. 

![image](https://cloud.githubusercontent.com/assets/6335546/14115464/20ef12a2-f5a9-11e5-95ce-c11069282e01.png)
![image](https://cloud.githubusercontent.com/assets/6335546/14115641/cb264a60-f5a9-11e5-8dc8-7eab6aeb4654.png)

I believe this gives us the benefit of using the "Required" label as we do in USDWS, making the label work for fixed with or variable width questions and draws attention to the label with by using red.
